conda/environments/cudf_dev_cuda11.5.yml

```
@@ -67,6 +67,13 @@ dependencies:
   - pydata-sphinx-theme
   - librdkafka=1.7.0
   - python-confluent-kafka=1.7.0
+  - moto>=3.1.6
+  - boto3>=1.21.21
+  - botocore>=1.24.21
+  - aiobotocore>=2.2.0
+  - s3fs>=2022.3.0
+  - flask
+  - flask_cors
```

This package name has a dash and not an underscore.
https://anaconda.org/search?q=flask-cors
https://pypi.org/project/Flask-Cors/
```suggestion
  - flask-cors>=3.0.0
```

---

removed, comment above.